apps/pie-storybook/stories/pie-tag.stories.ts

```diff
@@ -1,36 +1,118 @@
-import { html } from 'lit';
+import { html, nothing } from 'lit';
 
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-tag';
-import { TagProps } from '@justeattakeaway/pie-tag';
+import {
+    TagProps as TagBaseProps, variants, sizes,
+} from '@justeattakeaway/pie-tag';
 /* eslint-enable import/no-duplicates */
+import '@justeattakeaway/pie-icons-webc/IconHeartFilled';
 
-import { type StoryMeta } from '../types';
-import { createStory } from '../utilities';
+import type { StoryMeta, SlottedComponentProps } from '../types';
+import { createStory, type TemplateFunction } from '../utilities';
 
+type TagProps = SlottedComponentProps<TagBaseProps>;
 type TagStoryMeta = StoryMeta<TagProps>;
 
-const defaultArgs: TagProps = {};
+const defaultArgs: TagProps = {
+    variant: 'neutral',
+    size: 'large',
+    isStrong: false,
+    showIcon: false,
+    slot: 'Label',
+};
 
 const tagStoryMeta: TagStoryMeta = {
     title: 'Tag',
     component: 'pie-tag',
-    argTypes: {},
+    argTypes: {
+        variant: {
+            description: 'Set the variant of the tag.',
+            control: 'select',
+            options: variants,
+            defaultValue: {
+                summary: 'neutral',
+            },
+        },
+        size: {
+            description: 'Set the size of the tag.',
+            control: 'select',
+            options: sizes,
+            defaultValue: {
+                summary: 'large',
+            },
+        },
+        isStrong: {
+            description: 'If `true`, displays strong tag styles for some of the variant',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        showIcon: {
+            description: 'Enable to see the example of Tag with icon. Available only for large tag size.',
```

Comment:
nit, can be addressed in a follow up PR:
we probably need to mention `showIcon` is only used to control the story and not an actual prop.